### PR TITLE
completed css-brochure-form challenge -Bryan Brown

### PR DESCRIPTION
**Pull request by bbrown949**
Screenshot 2023-03-31 at 10 45 38 AM

### CODE REVIEW

File: css-brochure-form/index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  <h3>Learn More</h3>
+  <div>
+    <form>
+      <section>
+        <label for="email-address">COMPANY EMAIL</label>
+        <div id="email1">
+          <input name="email" type="email" id="email-address" placeholder="sally@company.co">
+        </div>
+  </div>
+  <div id="your-name">
```
Comment (shawnkost) [inline css-brochure-form/index.html]: This `id` should be getting applied to the `input` on line 28 instead

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change requested here Bryan!

File: css-brochure-form/index.html (hunk 1 above)
Comment (bbrown949) [reply css-brochure-form/index.html]: fixed!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**